git-credential: add page (#5115)
tldr-pages/tldr: pages/common/git-credential.md

# git credential

> Retrieve and store user credentials.
> More information: <https://git-scm.com/docs/git-credential>.

> - Display credential information, retrieving the username/password from config files:

Reviewer: Why not?
```suggestion
- Display credential information, retrieving the username and password from configuration files:
```

Author: It looks great! It's not actually my own wording, it's from [here](https://git-scm.com/docs/git-credential).

> `echo {{"url=http://example.com"}} | git credential fill`

Reviewer 2: I think the quotes should be outside the token syntax here - same goes for the other examples.

> - Send credential information to configured credential helpers to store for later use:

Reviewer 2: Perhaps this will read better?
```suggestion
- Send credential information to all configured credential helpers to store for later use:
```

....or would it change the meaning?

Author: No it wouldn't change the meaning.  They used the word "any" but I think "all" is better.

`echo {{"url=http://example.com"}} | git credential approve`

> - Send credential information to configured credential helpers to erase:

Reviewer 2: Perhaps this will read better?
```suggestion
- Erase the specified credential information from all the configured credential helpers:
```

...or would it change the meaning too much?

Author: Not at all.
